core/src/main/web/outputdisplay/bko-tabledisplay/bko-tabledisplay.js

```diff
@@ -191,9 +191,9 @@
               return;
             if (n.indexOf(suffix,n.length-suffix.length) === -1)
               n = n + suffix;
-            // TODO check for error
-            return bkHelper.saveFile(n,out);
-          } , "Select name for CSV file to save", "csv");
+            // TODO check for error, prompt for overwrite
+            return bkHelper.saveFile(n, out, true);
```

Comment:
best is to ask but that's more work than i have time for right now.
given the choice of silently overwrite or do nothing, overwrite is better.